packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -126,21 +138,41 @@ const PersonalDetails = ({
             validate={handleValidate}
             validateOnMount
             enableReinitialize
-            initialStatus={{ is_confirmed: !is_qualified_for_idv }}
+            initialStatus={{ is_confirmed: no_confirmation_needed || !is_qualified_for_idv }}
             onSubmit={(values, actions) => {
                 onSubmit(getCurrentStep() - 1, values, actions.setSubmitting, goToNextStep);
             }}
         >
-            {({ handleSubmit, errors, setFieldValue, touched, values, handleChange, handleBlur, status }) => (
+            {({
+                handleSubmit,
+                errors,
+                isSubmitting,
+                setFieldValue,
+                touched,
+                values,
+                handleChange,
+                handleBlur,
+                status,
+            }) => (
                 <AutoHeightWrapper default_height={380} height_offset={isDesktop() ? 81 : null}>
                     {({ setRef, height }) => (
                         <Form
+                            noValidate
                             ref={setRef}
                             onSubmit={handleSubmit}
                             autoComplete='off'
                             onClick={closeToolTip}
                             data-testid='personal_details_form'
                         >
+                            {should_scroll_to_error_field && (
+                                <ScrollToFieldWithError
+                                    fields_to_scroll_end={isMobile() ? '' : ['account_opening_reason']}
+                                    fields_to_scroll_top={isMobile() ? ['account_opening_reason'] : ''}
```

Comment:
please don't hard-code `account_opening_reason` in multiple places!
You can move it to a config file and access it from there!

Reply:
as it was discussed, we will refactor it in separate card, as all our fields names are currently used as just strings. so we should refactor it in many places